extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fade-in-scroll.js

```diff
@@ -59,29 +60,29 @@ config.run('amp-fx-collection', {}, function () {
         expect(getOpacity(win)).to.equal(0);
         win.scrollTo(0, 0.1 * getViewportHeight(win));
         return Promise.resolve()
-          .then(timeout(2000))
+          .then(timeout(ANIMATION_TIME))
           .then(() => {
             // Since the animation is spread over 50% of the viewport,
             // scrolling 10% of the viewport should change the opacity by 20%
             expect(getOpacity(win)).to.equal(0.2);
             win.scrollTo(0, 0.4 * getViewportHeight(win));
           })
-          .then(timeout(2000))
+          .then(timeout(ANIMATION_TIME))
```

Comment:
Should the `timeout: 20000` above be shorter as well?

Reply:
I'll just remove it so the default is used.